spec/models/control_spec.rb

```diff
@@ -42,17 +42,17 @@
     end
 
     it 'get low severity' do
```

Comment:
same as above, do we/should we have ones for 'none'/"Not Applicable" and Critical?